INSTALL.md

```diff
@@ -46,6 +52,12 @@ sudo dnf install -y \
     wget git
 ```
 
+For libsubid support:
+
+```sh
+sudo dnf --enablerepo=devel install shadow-utils-subid-devel
```

Comment:
I see that powertools is not enabled on the rockylinux:8 docker container, so it will need `--enablerepo=powertools`.   Also, since I asked you to add this to a BuildRequires in the rpm, in the INSTALL.md section about rpms add after where it says to install `system dependencies` the phrase `including for libsubid support`.

Reply:
@DrDaveD these changes should be all set. Let me know how things look.